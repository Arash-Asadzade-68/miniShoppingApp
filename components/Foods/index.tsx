import React from 'react';
import Styles from './styles.scss'
const Foods = () =>{
  return(
    <div className={Styles.headerContainer}>
        <h2>Foods</h2>
    </div>
  )
}

export default Foods;