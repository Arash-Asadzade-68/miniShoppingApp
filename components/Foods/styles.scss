.tabSvg {
  margin: -1px 0;
  fill: aliceblue;
  padding: 8px;
}