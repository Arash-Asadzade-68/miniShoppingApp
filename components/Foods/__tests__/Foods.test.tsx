import React from 'react';
import {shallow} from 'enzyme';
import ToJSON from 'enzyme-to-json'
import Foods from '../'

describe('Foods', ()=>{
  it('renders foods component',()=>{
    const wrapper = shallow(<Foods/>);
    expect(ToJSON(wrapper)).toMatchSnapshot(); 
  })
})