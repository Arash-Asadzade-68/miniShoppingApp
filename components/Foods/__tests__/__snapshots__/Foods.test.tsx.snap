// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Foods renders foods component 1`] = `
<div
  className="headerContainer"
>
  <h2>
    Foods
  </h2>
</div>
`;
